es/es-server/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -1018,69 +1029,53 @@ protected final IndexingStrategy planIndexingAsPrimary(Index index) throws IOExc
                 currentNotFoundOrDeleted = versionValue.isDelete();
             }
             if (index.getIfSeqNo() != SequenceNumbers.UNASSIGNED_SEQ_NO && versionValue == null) {
-                final VersionConflictEngineException e = new VersionConflictEngineException(
-                    shardId,
-                    index.id(),
-                    index.getIfSeqNo(),
-                    index.getIfPrimaryTerm(),
-                    SequenceNumbers.UNASSIGNED_SEQ_NO,
-                    0
-                );
-                plan = IndexingStrategy.skipDueToVersionConflict(e, currentNotFoundOrDeleted, currentVersion, getPrimaryTerm());
-            } else if (versionValue != null && index.getIfSeqNo() != SequenceNumbers.UNASSIGNED_SEQ_NO && (
+                final VersionConflictEngineException e = new VersionConflictEngineException(shardId, index.id(),
+                    index.getIfSeqNo(), index.getIfPrimaryTerm(), SequenceNumbers.UNASSIGNED_SEQ_NO,
+                    SequenceNumbers.UNASSIGNED_PRIMARY_TERM);
```

Comment:
nitpick: We usually try to have all arguments in a single line or have 1 line per argument. 

(There are a couple of other places as well where this has changed)